### DISCUSSION TITLE
Serverless how to include tesseract-js on my API route final build

### DISCUSSION THREAD

**Discussion by samuv** [vercel/next.js#13224] [Help]
Hi all,

I'm using nextjs targeting 'serverless' and I'm trying to find a solution for including a compiled library in my final bundle.

My problem is with a route API and the library tesseract-js where I need to specify a path for the file that was included in the node_modules but with the optimization and bundling in one file the library can't find it, I can specify a path for the file but I need to copy the compiled file on my final build

Any idea how can I achieve this?

**Comment by ggivingg** [comment]
+1
